### PR TITLE
#1920 - remove timeout logic for import and augment timeout

### PR DESCRIPTION
**Pull request by jcomedouteau**
- set timeout to 5mn
- Apply timeout only for download

### CODE REVIEW

File: shanoir-ng-front/src/app/shared/mass-download/mass-download.service.ts [hunk 1]
```diff
@@ -157,6 +159,8 @@ export class MassDownloadService {
         } else {
             directoryHandlePromise = this.getFolderHandle();
         }
+        let path :string = "Datasets-" + formatDate(new Date(), "yyyyMMddHHmmss", "en-US") + '/';
```
[the remark this answers is not in the document]
Comment (jcomedouteau) [reply shanoir-ng-front/src/app/shared/mass-download/mass-download.service.ts:162]: Ah yes, I did not think of this. Maybe you were right, this upper folder is not the greatest idea

File: shanoir-ng-front/src/app/shared/notifications/notifications.service.ts [hunk 2]
```diff
@@ -42,7 +42,7 @@ export class NotificationsService {
     readonly readInterval: number = 1000;
     readonly persistenceTime: number = 1800000;
     private freshTimeouts: SuperTimeout[] = [];
-    private readonly TIMEOUT: number = 30000;
+    private readonly TIMEOUT: number = 300000;
```
Comment (julien-louis) [inline shanoir-ng-front/src/app/shared/notifications/notifications.service.ts:45]: why ?

Comment (jcomedouteau) [reply shanoir-ng-front/src/app/shared/notifications/notifications.service.ts:45]: We had problem with the timeout, sometimes it takes more than 30sc to be updated.
The other problem was that this was applied for others jobs, taht weren't download, and import always finished as "timeout" before the end of he import

**Comment by jcomedouteau** [comment]
I'm closing this one, as this is integrated in Julien's PR.

**Pull request closed without merging**